modules/smart-agent_systemd-timers/conf/01-timer-has-not-run-for.yaml

```
@@ -0,0 +1,15 @@
+module: systemd
+name: "timer has not run for"
```

not a big fan of this name. a detector's name should ideally:
- describe the detector goal (ok here)
- be a complete sentence (ko here)
keep in mind that the name will be what is displayed on signalfx ui and in our doc like: https://github.com/claranet/terraform-signalfx-detectors/blob/feat-systemd-timers-detectors/docs/severity.md#systemd-timers

what about:
```suggestion
name: "execution delay"
```

This will create a detector with the name:
> [env] Systemd-timers execution delay
with alerts like:
> [env] Systemd-timers execution delay is too high (XX > YY)

If you want you can add a `description` attribute to the `rule` to customize the message append to detector's name like:
```hcl
rules:
  major:
    threshold: 1
    comparator: "<"
    description: "is too long"
    lasting_duration: 30m
```

---

I included your suggestion :checkered_flag: